packages/blocks/src/store/reducer.js

```diff
@@ -54,12 +79,7 @@ export function blockTypes( state = {}, action ) {
 		case 'ADD_BLOCK_TYPES':
 			return {
 				...state,
-				...keyBy(
-					map( action.blockTypes, ( blockType ) =>
-						omit( blockType, 'styles ' )
```

Comment:
There was a typo in the `styles ` string (see extra space at the end). Therefore styles were never removed. I removed this logic and left `styles` in the state for backward compatibility.